src/Xrefcheck/Scanners/Markdown.hs

```diff
@@ -44,6 +46,11 @@ defGithubMdConfig = MarkdownConfig
 instance Buildable Node where
   build (Node _mpos ty subs) = nameF (show ty) $ blockListF subs
 
+-- foldWithSpine :: Monoid a => (Node -> a) -> Node -> a
```

Comment:
I don't like this silent change. This commit is about removing mixing, but here you commented function for unknown reason